docs/software/other/mqtt.mdx

```diff
@@ -380,3 +380,34 @@ if __name__ == '__main__':
     while client.loop() == 0:
         pass
 ```
+### Tutorial on using MQTT with version 1.3.46+
+
+Version 1.3.46+ is the first 1.3 release with MQTT and JSON fully functional on supported devices. Heltec JSON is unlikely to work reliably.
```

Comment:
In Heltec devices, JSON over MQTT

Reply:
Absolutely. Thanks for the edit.  (I just got done testing the Heltec a few minutes ago and JSON over MQTT is inop but if it is turned off the protobuf method seems to work.)